### DISCUSSION TITLE
Add an optional to auto use /tpbypass

### DISCUSSION THREAD

**Discussion by sakurawald** [Zrips/CMI#7248] [Enhancement]
Add an optional to auto use /tpbypass.
i mean: if the teleportation destination is un-safe, auto choose the safe-location and teleport the player.
don't let the player to type /tpbypass command, it's too complex.

**Comment by mrfloris** [comment]
<img width="976" alt="Screen-Shot 2022-09-25 at 16 29 17" src="https://user-images.githubusercontent.com/28841349/192148952-e99258a4-3d38-49a0-821e-e8a2f5b554d7.png">
You can give the group or person the bypass permission for the bypass type. I think?

**Reply by sakurawald** [reply]
I have add this permission.
cmi.safeteleport.bypass.*
